src/client/components/pages/registration-auth.js

```diff
@@ -23,6 +23,13 @@ const Button = require('react-bootstrap').Button;
 const PageHeader = require('react-bootstrap').PageHeader;
 const Alert = require('react-bootstrap').Alert;
 
+/**
+* Renders a page allowing the user to sign in or register BookBrainz with the
```

Comment:
Right idea now - `to sign in to MusicBrainz to continue the BookBrainz registration process.` would make more sense.